x/staking/spec/delegation_sequence.puml

```diff
@@ -0,0 +1,104 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+title: Delegating (currently undelegated funds delegator)
+autonumber
+
+participant "msgServer (staking)"
+participant  "keeper (staking)" as keeper
+participant validator
+participant keeper.bankKeeper
+participant vestingAccount
+participant "func:SendCoins" as SendCoins
```

Comment:
The entity with which the flow is interacting has changed on this line. The function is now like an entity and I think it's better to keep entities at the type/module abstraction, e.g., keeper, so it's easier to identify where the call is being directed (the function call name can be on the call's line).

Reply:
Fixed, using `group` instead to distinguish the actions that take place in a specific function